<?xml version='1.0' ?>

<launch>

  <arg name="use_sim_time" default="true" description="Use the /clock topic for time to sync with simulation"/>
  <arg name="manager_address" default="localhost" description="The address for the manager"/>
  <arg name="manager_port" default="5002" description="The port for the manager"/>
  <arg name="output" default="screen"/>

  <!-- Door manager -->
  <node pkg="rmf_demos_door_adapter" exec="door_manager" output="both">
    <param name="use_sim_time" value="$(var use_sim_time)"/>
    <param name="manager_address" value="$(var manager_address)"/>
    <param name="manager_port" value="$(var manager_port)"/>
  </node>

  <!-- Door adapter -->
  <node pkg="rmf_demos_door_adapter" exec="door_adapter" output="both">
    <param name="use_sim_time" value="$(var use_sim_time)"/>
    <param name="manager_address" value="$(var manager_address)"/>
    <param name="manager_port" value="$(var manager_port)"/>
  </node>

</launch>
